tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -50,7 +50,10 @@ fi
 set +ex  # rvm script is very verbose and exits with errorcode
 source $HOME/.rvm/scripts/rvm
 set -e  # rvm commands are very verbose
-rvm use ruby-2.4
+rvm install 2.5.0
```

Comment:
how long does this take?   It will slow down all our mac builds. It might make sense to request installing ruby 2.5 by the kokoro team also so we don't have to do this forever.